src/sidebar/services/annotations.ts

```diff
@@ -337,4 +337,17 @@ export class AnnotationsService {
 
     return savedAnnotation;
   }
+
+  /**
+   * Reload an existing annotation details from the API, and update it to the
+   * store
+   */
+  async reloadAnnotation({ id }: SavedAnnotation): Promise<Annotation> {
```

Comment:
Nothing in the implementation enforces that the annotation must already be in the store. I think that's fine actually. This function could just be called `loadAnnotation` with a description "Fetch an annotation from the API and add or update it in the store."